src/ClientData/ScopeIdProviderTest.cpp

```diff
@@ -49,51 +53,56 @@ static void AssertNameToIdIsBijective(const std::vector<TimerInfo>& timers,
   }
 }
 
-static std::vector<uint64_t> GetIds(const std::vector<TimerInfo>& timers) {
+static std::vector<uint64_t> GetIds(const std::vector<orbit_client_protos::TimerInfo>& timers) {
   orbit_grpc_protos::CaptureOptions capture_options;
-  auto id_provider = NameEqualityApiEventIdProvider::Create(capture_options);
+  auto id_provider = NameEqualityScopeIdProvider::Create(capture_options);
 
   std::vector<uint64_t> ids;
   std::transform(std::begin(timers), std::end(timers), std::back_inserter(ids),
                  [&id_provider](const TimerInfo& timer) { return id_provider->ProvideId(timer); });
   return ids;
 }
 
-static void TestProvideId(std::vector<TimerInfo>& timer_infos) {
+static void TestProvideId(std::vector<orbit_client_protos::TimerInfo>& timer_infos) {
   AssertNameToIdIsBijective(timer_infos, GetIds(timer_infos));
 }
 
-TEST(NameEqualityApiEventIdProviderTest, ProvideIdIsCorrectForApiScope) {
+TEST(NameEqualityScopeIdProviderTest, ProvideIdIsCorrectForApiScope) {
+  absl::SetFlag(&FLAGS_devmode, true);
```

Comment:
Maybe add a Bug and refer it here (and below) with `// TODO(b/...): Remove as soon as feature is not hidden behind a flag`.

Reply:
And add the same todo to CmakeLists that add a dependency on ClientFlags?

Reply:
Yeah, probably.